earth_enterprise/src/scons/getversion.py

```diff
@@ -8,6 +8,8 @@
 import re
 
 
+
```

Comment:
Very minor: there's a couple of random extra newlines here. I'm fine with keeping them or removing them.